.github/workflows/ci-lite.yaml

```diff
@@ -154,6 +154,21 @@ jobs:
           cache-from: type=registry,ref=${{ needs.meta.outputs.container_base }}
           cache-to: type=inline
 
+  scan-docker-image-cves:
```

Comment:
don't requiring it as a dependency for CI/CD release, because for hotfixes it shouldn't block release